frontend/src/components/dialog/DialogBase.vue

```diff
@@ -0,0 +1,56 @@
+<script>
+export default {
+  name: 'DialogBase',
+  props: {
+    value: { default: null, required: true }
+  },
+  data () {
+    return {
+      entityProperties: [],
+      entityData: {},
+      entityUri: '',
+      visible: false
+    }
+  },
+  watch: {
+    visible: function (visible) {
+      if (!visible) {
+        this.$emit('input', null)
+      }
+    },
+    value: function (id) {
+      this.visible = (!!id)
+    }
+  },
+  methods: {
+    clearEntityData () {
+      this.setEntityData({})
+    },
+    loadEntityData (uri) {
+      this.clearEntityData()
+      if (uri) {
+        this.entityUri = uri
+        this.api.get(uri)._meta.loaded.then(this.setEntityData)
```

Comment:
`loaded` wurde vor Kurzem umbenannt in `load`. Gibt sonst einen Konsolenfehler.
```suggestion
        this.api.get(uri)._meta.load.then(this.setEntityData)
```

Reply:
einverstanden